fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MvUtils.java

```diff
@@ -469,6 +470,23 @@ public Void visitLogicalJoin(OptExpression optExpression, Void context) {
         return joinExprs;
     }
 
+    public static void splitPredicate(
+            Set<ScalarOperator> predicates,
+            Set<String> equivalenceColumns,
+            Set<String> nonEquivalenceColumns) {
+        for (ScalarOperator operator : predicates) {
+            if (!MVUtils.isPredicateUsedForPrefixIndex(operator)) {
+                continue;
+            }
```

Comment:
What's the differece between `isPredicateUsedForPrefixIndex ` and `isEquivalencePredicate `

Reply:
isPredicateUsedForPrefixIndex include  not equal binary predicate like a > 1;
while isEquivalencePredicate include only in predicate or equal binary predicate like a = 1